packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -176,11 +175,21 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
   _captureSession = [[AVCaptureSession alloc] init];
   AVCaptureSessionPreset preset;
   if ([resolutionPreset isEqualToString:@"high"]) {
-    preset = AVCaptureSessionPreset1280x720;
-    _previewSize = CGSizeMake(1280, 720);
+    if ([_captureSession canSetSessionPreset:AVCaptureSessionPreset1920x1080]) {
+      preset = AVCaptureSessionPreset1920x1080;
+      _previewSize = CGSizeMake(1920, 1080);
+    } else {
+      preset = AVCaptureSessionPreset1280x720;
```

Comment:
Do we need to potentially bail down to lower resolutions in these fallback cases too? Is it possible that we won't be able to set a preset at this slightly lower size either?